README.md

```diff
@@ -87,3 +87,11 @@ The primary supported way of running PeopleMath in a staging or production confi
 Authentication using Firebase is in development.
 Currently, only the frontend is implemented. As long as
 `requireAuth` is set to `false` in `src/environments/environment.prod.ts` the app will run as expected without authentication.
+
+To enable authentication using Firebase:
+
+* Set `requireAuth` to `true` in `src/environments/environment.prod.ts`
+* Download your [firebase config file](https://support.google.com/firebase/answer/7015592#web) and copy the text into the `firebaseConfig.ts` file in the same folder.
```

Comment:
Nit: `Firebase` (capital F)

Same on the line below.

Reply:
Resolved.